### ISSUE TITLE
Monorepo Progress Updates: Day N+2

### ISSUE THREAD

**Issue by jtmcdole** [flutter/flutter#160628]
Hello Flutter Community!

**We will use this issue to communicate further progress.**

**Day 1** saw the merging of three repositories and then some fast-follow updates to kick the tires on presubmits and the Merge Queue building production artifacts.

**Day 2** was landing quick updates to re-add some missing files and make our update scripts aware of[ multiple origins](https://github.com/flutter/flutter/issues/160558). It was rough going through the MQ as our PR, which passed all pre-submits, was blocked by infrastructure failures. We were sure our changes would work and so we guided the PR through CI till it landed.

**Day N+2**: You are here
* We've fixed some scheduling errors on the dashboard - backfilling appears to be working.
* We've landed our format-the-world PRs
* **The tree is open.**

If you have a PR that was created **before the merge**, you will run into some problems. We believe these are related to the PR structure on GitHub and are investigating. The work around is:
* Create a new PR after the monorepo merge
* Run dart format over your changed files
* Now's a good time to turn on "auto format on save" in your editor of choice.

If you have any problems with presubmit tests, the merge queue, or generally about the monorepo; please [file an issue and use the <monorepo> label](https://github.com/flutter/flutter/issues/new?labels=monorepo).

**Comment by orestesgaolin** [comment]
Quick question on docs -- will they be migrated to flutter/flutter from [flutter/engine/docs](https://github.com/flutter/engine/tree/main/docs)?

**Comment by jtmcdole** [comment]
**Update**

There is a known race condition in the MQ when building assets which can sometimes lead to the wrong assets being build. See #160704 for more information.
